src/cpp/cpp_scope.cpp

```diff
@@ -143,7 +143,8 @@ void cpp_scopet::lookup(
   }
 
   if(!id_set.empty() &&
-     id_class!=id_classt::TEMPLATE) return; // done, upwards scopes are hidden
+     id_class!=id_classt::TEMPLATE)
+    return; // done, upwards scopes are hidden
```

Comment:
Nit: since the if statement is multiple lines, wrap return in braces ( @peterschrammel  )

Reply:
Moved if condition to single line in [PR #6](https://github.com/tautschnig/cbmc/pull/6)